src/js/engagement_view/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "grapl-fe",
   "version": "0.1.0",
   "private": true,
+  "engines": {
```

Comment:
This sets a warning if the wrong version of node is used

Reply:
Looks good, wondering if you can take a look at the last line of `src/js/engagement_view/package.json`. When `packageManager: yarn@3.1.1.` is removed, @cm and I noticed the project still builds with tests passing locally, wondering if this is necessary?

Reply:
So the engines section is not necessary since it will only warn not error out.
The packageManager setting is necessary since that's what corepack uses to figure out the yarn version. Without that set, you'll use whatever the default yarn version is for your system. Since I set up yarn 3.1.1 with you manually, it's probably using the correct version. Alternatively, it may just build correctly even with yarn 1.x.x